e2e/discussions.spec.ts

```diff
@@ -0,0 +1,465 @@
+import { Page, expect, test } from '@playwright/test'
+import { testAuth } from './fixtures/authenticatedUserPages'
+import { entityUrlPathname } from './helpers/entities'
+import {
+  setupProjectWithPermissions,
+  teardownProjectsAndFileHandles,
+} from './helpers/setupTeardown'
+import { dismissAlert } from './helpers/testUser'
+import { Project } from './helpers/types'
+import { UserPrefixes, userConfigs } from './helpers/userConfig'
+
+const discussionActionIconClasses = {
+  PIN: '.fa-thumb-tack.imageButton',
+  EDIT: '.fa-pencil.imageButton',
+  DELETE: '.syn-trash-o.imageButton',
+  TOGGLE_FOLLOW: '.fa-eye.imageButton',
+  LINK: '.fa-link.imageButton',
+}
```

Comment:
Discussion action icons are not buttons and don't have accessible names, so can't be selected by role + name or by label -- select by classname to differentiate between the icons.

Reply:
That's not good! This is fine for now. Really like that RTL/Playwright basically force you to do surface level a11y testing. Do you mind adding your note as a comment in a `// FIXME`?

Reply:
Good call! Done in the next commit